src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,44 +1,110 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 10/14/17
  */
+import com.cs56fitnessapp.models.workout.Workout;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Date;
+
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** date */
+    private LocalDate date;
+
+    /** User */
+    private User user;
+
+    /** calorieGoal */
+    private double calorieGoal;
+
+    /** WorkoutList */
+    private ArrayList < Workout > workoutList;
+
+    /** MealList */
+    private ArrayList < Meal > mealList;
+
+    public Day (LocalDate date, User user){
+      this.date = date;
+      this.user = user;
+    }
+
+
+    public boolean isToday() {
+        LocalDate now = LocalDate.now();
+
+        return now.getYear() == date.getYear() && now.getMonth() == date.getMonth() && now.getDayOfMonth() == date.getDayOfMonth();
+    }
+
+    // TODO:
+    // Kcal ~= METS * bodyMassKg * timePerformingHours
+    public double getCaloriesLeft() {
+        return 0;
+    }
 
     /* ActivityFacts Interface */
     @Override
     public int getCaloriesOut() {
-        return 0;
+        int sum = 0;
+        for (int i = 0; i < workoutList.size(); ++i) {
+            Workout workout = workoutList.get(i);
+            sum += workout.getCaloriesOut();
+        }
+        return sum;
     }
 
     @Override
-    public int getActiveTimeMins() {
+    public int getActiveTimeMins()
+    {
         return 0;
     }
 
```

Comment:
Calculate total active minutes during the . day, in the same spirit that your calculated `getCaloriesOut()`
Use `workoutList.get(i).getActiveTime()`.